recipe/meta.yaml

```diff
@@ -146,7 +146,7 @@ outputs:
       build:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
-        - sysroot_linux-64 2.17  # [linux]
+        - sysroot_linux-64 2.28  # [linux]
```

Comment:
We don't have the RHEL 8 sysroot available yet: https://github.com/conda-forge/conda-forge.github.io/issues/1941